shared/cfg/guest-os/Linux/Fedora/28.cfg

```diff
@@ -40,7 +40,7 @@
             # but fails ("No space left on device") with methods such as url.
             mem = 2048
             # Unless overridden use secondary url, because it's the most common one
-            url = http://dl.fedoraproject.org/pub/fedora-secondary/releases/28/Server/${vm_arch_name}/os/
+            url = https://archives.fedoraproject.org/pub/archive/fedora-secondary/releases/28/Server/${vm_arch_name}/os/
```

Comment:
this file should be changed in a different commit as it is not related to F31 at all.

hint: If you need to use "and/also/additionally" in main commit message it usually indicates you are doing multiple things and you should perhaps split... ;-)

Reply:
Thanks for your useful hint,will keep it on mind in the future.